lib/auth/auth.go

```diff
@@ -925,6 +934,12 @@ func (a *Server) runPeriodicOperations() {
 	})
 	defer instancePeriodics.Stop()
 
+	ossDesktopsCheck := interval.New(interval.Config{
```

Comment:
If we know this is enterprise, can we skip this check completely?

Reply:
Yes, I've changed it.